src/index.html

```diff
@@ -17,11 +17,410 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The `href` attribute in the link tag should point to a CSS file, not an SCSS file. SCSS files need to be compiled into CSS before they can be used in an HTML document.